inlong-manager/pom.xml

```diff
@@ -129,6 +129,11 @@
             <version>${h2.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
The manager module should not depend on the agent module.
It's recommended depending on the inlong-common module.